sound/effect.h

```diff
@@ -423,13 +423,24 @@ class Effect {
       }
 
 #ifdef ENABLE_AUDIO
+#if VERSION_MAJOR < 4
       else if (strlen(dir) > 8) { // TODO: Check individual path segments
+        STDOUT << "\nFont directory not found\n";
+        STDOUT << " and name exceeds 8 characters: " << dir << "\n";
```

Comment:
Just do it like:

STDOUT << "line one\n"
                      "line two: " << dir << "\n"

Reply:
That's what I was saying about the new line confusion. 
Without a new line at the front end, it tacks on to the end previous printout like
```
Scanning sound font: TeensySFFont directory not found: TeensySF
```

Reply:
That just means that the "done" should be moved up.

Reply:
Couldn't get just moving "done" to work because then all the scanned fonts had no line return, but duplicating does.
Interested what's up with the "new syntax" for the message lines as opposed to println. Would that not have been easier or is it now improper?